docs/en/README.md

```diff
@@ -74,3 +74,23 @@ cases:
 actionText: Quick Start →
 actionLink: /en/installing/compilation
 ---
+
```

Comment:
Do we need to add the License header on the markdown file?

Reply:
We have added a license header to all markdown files.
But this file is special and needs to be rendered as the homepage of the Doris website through `Vuepress`.
We have tried to add the license, but that will cause the rendering to fail.
Does this need to be specifically stated in the LICENSE file?

Reply:
Oh, I can put the license in the `frontmatter` section. I will add it